packages/utilities-react/src/StepFlow/StepGroup.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright IBM Corp. 2025, 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useEffect } from 'react';
+import { useStepContext } from '.';
+
+interface StepGroupProps {
+  children: React.ReactElement;
+}
+
+/**
+ * Container for steps, each child represents a step
+ */
+export const StepGroup: React.FC<StepGroupProps> = ({ children }) => {
+  const { setTotalSteps, currentStep } = useStepContext();
+
+  const childrenArray = React.Children.toArray(children);
+  const cleanedChildren = childrenArray?.filter(Boolean);
```

Comment:
I see an optional chaining operator here. When will `childrenArray` not be an array?